src/main/resources/openapi.json

```diff
@@ -1462,6 +1499,61 @@
                         "topic2"
                     ]
                 }
+            },
+            "AssignedTopicPartitions": {
+                "title": "AssignedTopicPartitions",
+                "type": "object",
+                "properties": {
+                    "topic": {
```

Comment:
I still don't understand one thing. From this declaration it seems that this object has to have a field named "topic" which is not the case, or?

Reply:
```"partitions": [
        {
            "test": [
                3,
                4
            ]
        }
    ]```
From this should be clear the topic `test` has partitions 3 and 4.

Reply:
I was really not convinced about that declaration because it clearly says that the JSON can have a field "topic" which is an array of int and it's not mandatory. I had a chat with a colleague with more OpenAPI knowledge and the outcome is the following. 
In our case, the example with "topic1" is working just because we are leveraging the `additionalProperties` field which is by default always true. It means that the OpenAPI validation just considers "topic1" as an additional property and not as the "topic" field mentioned above. The right specification is to explicit the `additionalProperties` defining them as arrays of int so something like this:

```yaml
title: AssignedTopicPartitions
type: object
additionalProperties:
        type: array
        items:
            format: int32
            type: integer
```

This is the YAML, you should change it to JSON.
Can you try this one but even having a more complex example with more arrays, so for example ...

```json
{
    "topic1": [
        1,
        2,
        3
    ],
    "topic2": [
        1,
        2
    ]
}
```